src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -79,7 +79,7 @@ const WorkspaceTabs = ({
 
   const tabs = [
     { name: 'dashboard', link: 'workspace-dashboard' },
-    ...(isGoogleWorkspace ? [{ name: 'data', link: 'workspace-data' }] : []),
+    ...([{ name: 'data', link: 'workspace-data' }]),
```

Comment:
I realize this wasn't part of the Jira ticket (my bad!), but now that I see it in code: how about replacing `isGoogleWorkspace` with `isFeaturePreviewEnabled('workspace-data-service')`? That way, we'd only show the Data tab for Azure workspaces if the user has the feature flag enabled.